### DISCUSSION TITLE
movie extras cleanup?

### DISCUSSION THREAD

**Discussion by quadcom** [RandomNinjaAtk/docker-radarr-extended#51] [Q&A]
I've turned off Kodi compatibility to gain the subfolder to house all the extras media. But, the library currently has extras in the Kodi format.

Will the daily autoextras script work with the movieextras to clear up the old stuff and replace it with the new media? I went through the code and I thought I saw something in there regarding deleting stuff and replacing it.

**Comment by RandomNinjaAtk** [comment]
Pretty sure it doesn't do that.
